controllers/appwrapper_controller.go

```diff
@@ -175,99 +189,32 @@ func (r *AppWrapperReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
-func addAppwrappersThatNeedScaling() {
-	kubeconfig := os.Getenv("KUBECONFIG")
-	restConfig, err := getRestConfig(kubeconfig)
-	if err != nil {
-		klog.Fatalf("Error getting config: %v", err)
-	}
-	awJobClient, _, err := clients.NewClient(restConfig)
-	if err != nil {
-		klog.Fatalf("Error creating client: %v", err)
-	}
-	queueJobInformer := arbinformers.NewSharedInformerFactory(awJobClient, 0).AppWrapper().AppWrappers()
-	queueJobInformer.Informer().AddEventHandler(
-		cache.FilteringResourceEventHandler{
-			FilterFunc: func(obj interface{}) bool {
-				switch t := obj.(type) {
-				case *arbv1.AppWrapper:
-					klog.V(10).Infof("[getDispatchedAppWrappers] Filtered name=%s/%s",
-						t.Namespace, t.Name)
-					return true
-				default:
-					return false
-				}
-			},
-			Handler: cache.ResourceEventHandlerFuncs{
-				AddFunc:    onAdd,
-				UpdateFunc: onUpdate,
-				DeleteFunc: onDelete,
-			},
-		})
-	queueJobLister = queueJobInformer.Lister()
-	queueJobSynced := queueJobInformer.Informer().HasSynced
-	stopCh := make(chan struct{})
-	defer close(stopCh)
-	go queueJobInformer.Informer().Run(stopCh)
-	cache.WaitForCacheSync(stopCh, queueJobSynced)
-	<-stopCh
-}
-
-// onAdd is the function executed when the kubernetes informer notified the
-// presence of a new kubernetes node in the cluster
-func onAdd(obj interface{}) {
-	aw, ok := obj.(*arbv1.AppWrapper)
-	if ok {
-		klog.Infof("Found Appwrapper named %s that has status %v", aw.Name, aw.Status.State)
-		if aw.Status.State == arbv1.AppWrapperStateEnqueued || aw.Status.State == "" {
-			//scaledAppwrapper = append(scaledAppwrapper, aw.Name)
-			demandPerInstanceType := discoverInstanceTypes(aw)
-			//TODO: simplify the looping
-			if useMachineSets {
+/*
```

Comment:
This is a pretty big commented block. Why are we leaving it in?

Reply:
I forgot to remove it I'll leave it out